### PR TITLE
Adding basic support for Symertricom-Microsemi SSU2000

### PR DESCRIPTION
**Pull request by craig-nokia**
Add basic support for the original Symetricomm SSU2000 appliance. Company was sold to Microsemi.

Current support is for Stratum Clocks, GPS, PTP and NTP modules only.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/ssu2000.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+os: ssu2000
+text: 'Microsemi Synchronization Supply Unit'
+type: appliance
+icon: microsemi
+mib_dir: microsemi
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.601.3.1.
+discovery_modules:
```
Comment (PipoCanaja) [inline includes/definitions/ssu2000.yaml:10]: Hi @craig-nokia 
Any good reason to disable manually so many discovery modules ?

Comment (craig-nokia) [reply includes/definitions/ssu2000.yaml:10]: Hi PipoCanaja,

The platform is just a bunch of specialized modules running under a communication processor.  I would guess for specialized hardware support, there is no reason to take resources for unnecessary discovery/polling.  I don't want to hammer the device  :)

Comment (PipoCanaja) [reply includes/definitions/ssu2000.yaml:10]: Usually, a discovery (when not supported by the device) only costs 1 or 2 snmp requests ... So you could very well leave it open, unless the SNMP support of the device is suffering from it (like an SNMP server that would crash if you request a non existing OID. ).
So you could probably remove that code without any impact on the device nor on the discovery/polling time.

Comment (craig-nokia) [reply includes/definitions/ssu2000.yaml:10]: It's not a big deal to leave as I'm just trying to keep unnecessary traffic off the 15+ year old device.  I think I'm the only one  using this platform with librenms.    BTW, I did this same thing for the GE Pulsar power system several weeks ago.

Comment (craig-nokia) [reply includes/definitions/ssu2000.yaml:10]: BTW, it was the polling that hammers.  Not the discovery.

**Review by Jellyfrog — APPROVED** [review]

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-68-release-changelog-september-2020/13525/1

**Pull request merged**